source/develop/release-management/features/network/spikes/ost_net_failures_2019.html.md

```diff
@@ -0,0 +1,94 @@
+# Inconsistent but repeated failures are happening on OST 4.3 and master
+
+
+### Discussion
+
+Until now the policy for handling network requests in engine was:
+1. REST API requests are handled async unless the underlying commands report their Vdsm Tasks.
+2. <code>PersistentHostSetupNetworksCommand</code> flows are run async internally in the server.
+3. an external network action is revoked if the server is busy (lock taken)
+4. an internal refresh host action waits for a lock forever
+
+Policy (3) guarantees execution of requests in the order that they were issued - whether from a single client or from
+multiple clients. It also did not fail the 'human admin' usage model where a person can retry the request and check for
+its successful completion.
+However it does not perform well in a testing environment where revocations of requests due to 'busy server' fail the
+suite frequently.
+
+
+An alternate server policy for test suites must fulfill the following requirements:
+1. execution of requests in the same order that they were requested by a single client
+2. the server cannot be respond as 'busy' - it must queue external requests if it cannot serve them
+
+To fulfill requirement (1) it is necessary to make the REST API requests return synchronously with the completion of their
+operations.
+
+To fulfill requirement (2) a timed-wait for setup-networks was introduced to master. It helped reduce the failures on master
+considerably.
+The current failures on master all are around a similar situation with an external refresh-caps being revoked on 'server
+busy'. This situation might be resolved with a timed-wait for refresh-caps (https://gerrit.ovirt.org/#/c/98257).
+
+
+### OST Failures on branch 4.3
+
+  * 145 Fault detail is "[]"
```

Comment:
Can you please link the specific pages here? this way ost_net_failures_2019.html.md could be an entry point to the topic.